src/views/dependencyExplorer.ts

```diff
@@ -49,7 +50,6 @@ export class DependencyExplorer {
         if (!current) {
             return;
         }
-
```

Comment:
Avoid the empty diff if possible. You can always provide another PR for format/coding convention to avoid distraction.